src/main/java/org/openhab/automation/jrule/internal/engine/timer/TimerExecutor.java

```diff
@@ -47,9 +47,9 @@ public void add(JRuleTimedCronExecutionContext executionContext) {
     }
 
     public void add(JRuleTimeTimerExecutionContext executionContext) {
-        String cron = String.format("%s %s %s %s %s %s", executionContext.getSecond().map(String::valueOf).orElse("*"),
-                executionContext.getMinute().map(String::valueOf).orElse("*"),
-                executionContext.getHour().map(String::valueOf).orElse("*"), "*", "*", "*");
+        String cron = String.format("%s %s %s %s %s %s", executionContext.getSecond().map(String::valueOf).orElse("0"),
+                executionContext.getMinute().map(String::valueOf).orElse("0"),
+                executionContext.getHour().map(String::valueOf).orElse("0"), "*", "*", "*");
```

Comment:
Why use %s with the last 3 static fields? `* * *`?

I find the code reasier to read the fewer replacements there are.

Reply:
I wasn't sure if i should add flags for day, month, day-of-week to JRuleWhenTimeTrigger as well. This is the preparation for this. I could remove the "unused" %s flags. Shall I?

Reply:
@seaside1  wdyt?

Reply:
So, I guess it is easier to have fewer replacements, but then again if you plan on adding flags for day, month etc. Then maybe not worth changing back?